test/external/fuzz_linearizer.py

```diff
@@ -76,6 +76,11 @@ def fuzz_linearizer(lin: Linearizer):
   last_lins = [lin]
   failures = defaultdict(list)
 
+  FUZZ_MAX_SIZE=getenv("FUZZ_MAX_SIZE", 0)
```

Comment:
nit: can you fix the spaces between `=` and `>`? (this block and line 74)

Reply:
done and fixed line 80 too